### PR TITLE
LPS-64229 Fixes NPE when adding a Password Policy

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-64229](https://issues.liferay.com/browse/LPS-64229).

@JorgeFerrer: The last change ended up causing a regression (contrary to my assurance that "it will probably not cause any regressions"). Sorry about that.

@xbrianlee @alee8888 I'm curious why [all the tests passed](https://github.com/JorgeFerrer/liferay-portal/pull/688#issuecomment-197062825). Any ideas?
<br>/cc @arbetts @pei-jung

### CODE REVIEW

**Comment by JorgeFerrer** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/foundation/password-policies-admin/password-policies-admin-web/bnd.bnd [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Bundle-Name: Liferay Password Policies Admin Web
 Bundle-SymbolicName: com.liferay.password.policies.admin.web
 Bundle-Version: 1.0.0
+Include-Resource: @com.liferay.portal.impl-*.jar!/com/liferay/portal/model/PasswordPolicyImpl.class,\
+
```
Comment (JorgeFerrer) [inline modules/apps/foundation/password-policies-admin/password-policies-admin-web/bnd.bnd]: Hey @drewbrokke,

I don't think this is a good practice. 
@rotty3000 Do you think this is acceptable in order to be able to do new PasswordPolicyImpl() in the JSP for the form?
If not, one solution I can think of is to create a custom empty impl of PasswordPolicy. Can you think of something better?

@drewbrokke if Ray happens to think this is acceptable go ahead and send the pull to Brian.

**Comment by JorgeFerrer** [comment]
Closing for now. See comment in the code.

File: modules/apps/foundation/password-policies-admin/password-policies-admin-web/bnd.bnd (hunk 1 above)
Comment (drewbrokke) [inline modules/apps/foundation/password-policies-admin/password-policies-admin-web/bnd.bnd]: @JorgeFerrer @rotty3000 I found a better way than this.  I'll resend to you, Jorge.

**Comment by drewbrokke** [comment]
Pr sent: jorgeferrer#691

**Pull request closed without merging**